cpp/src/parquet/encoding_test.cc

```diff
@@ -1493,5 +1528,105 @@ TYPED_TEST(TestDeltaBitPackEncoding, NonZeroPaddedMiniblockBitWidth) {
   }
 }
 
+// ----------------------------------------------------------------------
+// DELTA_LENGTH_BYTE_ARRAY encode/decode tests.
+
+template <typename Type>
+class TestDeltaLengthByteArrayEncoding : public TestEncodingBase<Type> {
+ public:
+  using c_type = typename Type::c_type;
+  static constexpr int TYPE = Type::type_num;
+
+  virtual void CheckRoundtrip() {
+    auto encoder =
+        MakeTypedEncoder<Type>(Encoding::DELTA_LENGTH_BYTE_ARRAY, false, descr_.get());
```

Comment:
What is false here? Please add parameter name for clarity:
```suggestion
        MakeTypedEncoder<Type>(Encoding::DELTA_LENGTH_BYTE_ARRAY, /*xxx=*/false, descr_.get());
```

Reply:
Added.